i18n.go

```diff
@@ -26,43 +24,50 @@ func newI18n(opts ...Option) {
 		ins.getLngHandler = defaultGetLngHandler
 	}
 
-	atI18n = ins
+	return ins
 }
 
 // Localize ...
 func Localize(opts ...Option) gin.HandlerFunc {
-	newI18n(opts...)
+	atI18n := newI18n(opts...)
 	return func(context *gin.Context) {
+		context.Set("i18n", atI18n)
 		atI18n.setCurrentContext(context)
```

Comment:
@tung491 You should remove this line of code, because of it is no longer available.